app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -44,41 +44,57 @@ import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.constraintlayout.compose.ConstraintLayout
 import com.github.se.eventradar.R
+import com.github.se.eventradar.model.Location
 import com.github.se.eventradar.model.event.Event
 import com.github.se.eventradar.model.event.EventCategory
+import com.github.se.eventradar.model.event.Ticket
 import com.github.se.eventradar.ui.BottomNavigationMenu
 import com.github.se.eventradar.ui.navigation.TOP_LEVEL_DESTINATIONS
-import java.time.LocalDate
-import java.time.LocalTime
+import java.time.LocalDateTime
 
 @Composable
-fun HomeScreen() {
+fun HomeScreen() { // viewModel: EventViewModel = viewModel() + Navigation
```

Comment:
Has this been synced with main? There should be a navigation actions and view model in the parameters.